.github/workflows/check.yml

```diff
@@ -14,5 +14,6 @@ jobs:
         with:
           node-version: 14.x
       - run: npm ci
+      - run: npm run build
```

Comment:
Adding this here doesn't really solve the problem, someone running `grunt check-closure-compiler` locally would still have the problem. Need to make `check-closure-compiler` run `build` first -- add it as the first element of the array here https://github.com/ably/ably-js/blob/main/Gruntfile.js#L95